### PR TITLE
Add temporary limits to 2 wt

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

**Comment by Meklo** [comment]
I tried to improve coverage by every mean possible, @EqualsAndHashCode needed to be added to CurrentLimitsData and TemporaryLimitData in order to prevent get method to return the same 2WT on a substation because the method was unable to compare limits data. 
Unfortunaly i'm unable to test every cases added by @EqualsAndHashCode and since there is only little code added outside of the test package, it really tanks my coverage.

File: src/main/java/org/gridsuite/network/map/model/CurrentLimitsData.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import lombok.Builder;
+import lombok.EqualsAndHashCode;
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/network/map/model/CurrentLimitsData.java:11]: just curious: why did you need EqualsAndHashCode ?

**Review by dbraquart — COMMENTED** [review]
tests: ok
review: ok, just a question

File: src/main/java/org/gridsuite/network/map/model/CurrentLimitsData.java (hunk 1 above)
Comment (dbraquart) [reply src/main/java/org/gridsuite/network/map/model/CurrentLimitsData.java:11]: ok thanks!

**Review by dbraquart — APPROVED** [review]
